Création du manuel en français et traduction en anglais. (#9)
linuxmao-org/gmajctl: docs/manual/en/manual.txt

// -*- mode: adoc; -*-

:lang: en
:toc-title: Table of content
:last-update-label: Last update

> :warning-caption: Attention

Reviewer: Ces 3 lignes au-dessus peuvent être enlevées.
Ces éléments ont par défaut une définition en anglais : `toc-title`, `last-update-label`, `warning-caption`.

Author: Supprimé.

= User manual
Julien Taverna, Jean Pierre Cimalando and Olivier Humbert
v0.1
:toc:

[.Lead]

> This manual documents the use of FreeMajor program, a MIDI controler for TC Electronic G-Major multi-effect. +

Reviewer: Plusieurs corrections.

```suggestion
This manual documents the use of FreeMajor program, a MIDI controller for the TC Electronic G-Major multi-effects unit. +
```

> FreeMajor is a free and open source software. +

Reviewer: Pas d'article `a` pour [software](https://english.stackexchange.com/questions/12990/why-dont-we-use-the-indefinite-article-with-software).
`is a free and open source software` → `is free and open source software`

Author: Corrigé.

> If the software displays a problem, please post an issue on project page&nbsp;: +

Reviewer: ```suggestion
If the software displays a problem, please post an issue on the project page&nbsp;: +
```

> If the software présents a problem, please post an issue on project page&nbsp;: +

Reviewer: `présents` :smile: 
Je pense à `displays` qui convient peut être mieux à la phrase.

Author: Effectivement :-)
Corrigé.

https://github.com/linuxmao-org/FreeMajor

== Description

> FreeMajor is a MIDI controler compatible with TC Electronic G-Major multi-effect.

Reviewer: ```suggestion
FreeMajor is a MIDI controller compatible with the TC Electronic G-Major multi-effects unit.
```

> This software makes the advanced use of multi-effect easy to use thanks to a graphical interface that groups all the parameters of the presets.

Reviewer: ```suggestion
This software makes advanced use of the multi-effects processor easy, thanks to a graphical interface that groups all the parameters of the presets.
```

It offers the possibility of retrieving user presets from the G-Major, creating new ones, modifying existing ones, importing them, exporting them ...

You will discover all the possibilities offered by FreeMajor by reading this manual.

== Installation

> The project provides downloads for the Windows system. +

Reviewer: ```suggestion
The project provides downloads for Windows and macOS systems. +
```

https://github.com/linuxmao-org/FreeMajor/releases

> If you do not have a downloadable version for your operating system, you will need to recover the source code and build it yourself.

Reviewer: - `do not have`

À mon avis `have` dérange car il exprime la *possession* de la chose, non le fait d'*avoir à disponibilité*.
Éventuellement j'écrirais plutôt : `Unless there exists a downloadable version for your operating system,`.

- `recover` → `obtain`

Usage douteux de `recover` à mon avis. Il y a dans `recover` l'idée de récupération de quelque chose de perdu.

Author: Corrigé.

> Visit the project home page and follow the construction steps in the file README.md.

Reviewer: ```suggestion
Visit the project home page and follow the build steps in the file README.md.
```

== How to use

When the controller is opened, the window displays the graphical interface where you can find all the settings that are physically available on the G-Major and the functions needed to transfer the data to the G-Major.

.Edition window
image::../resources/en/screen.png[500,500]

The layout of the controller is divided into two parts&nbsp;:

The first part at the top of the user interface groups&nbsp;:

* Patches

image::../resources/en/presets.png[presets,357,105]

* MIDI

image::../resources/en/midi.png[midi,316,73]

* Patch parameters

image::../resources/en/parameters.png[parameters,586,100]

> * Parameter infos

Reviewer: `infos` → `information`
`information` ne se met pas au pluriel, pareil que `software`.

Author: Corrigé.

image::../resources/en/infos.png[infos,321,110]

The second part at the bottom of the user interface groups the effects&nbsp;:

- Noise Gate

- Compressor

- Equalizer

- Filter

- Pitch

- Chorus / Flanger

- Delay

- Reverb

> The use of different algorithms is documented in the G-Major manual. It is for this reason that we will not return here.

Reviewer: À reformuler. Par exemple :

```suggestion
The detailed use of each algorithm is already documented in the G-Major manual. We do not cover this topic in this document.
```

=== Patches

> Selection Window - Select a preset on which you want to work in the list of presets. If the real-time option is activated, the preset will be sent to the G-Major who will load this patch into the user bank and change the preset to use it.

Reviewer: ```suggestion
Selection Window - Select a preset from the list on which you want to work. If the real-time option is active, the preset is loaded into the user bank of the G-Major, and it is made the active program.
```

Load - Loads a previously saved .realmajor preset bank file. This resets FreeMajor and displays only the presets contained in the loaded bank.

Save - Saves all the presets being edited in FreeMajor to a preset bank file .realmajor.

> New - Creates a new empty preset in the selected position. If a preset is present on this location it is destroyed.

Reviewer: ```suggestion
New - Creates a new empty preset at the selected position. If a preset is already present at this location, it is destroyed.
```

> Change - If the real-time option is off, this button changes the G-Major preset.

Reviewer: - `If the real-time option is off`
C'est inexact. Il change le programme dans tous les cas, peu importe que l'option soit *on* ou *off*.

- `changes the G-Major preset`
Il change quoi exactement ? pas clairement expliqué je trouve.

```suggestion
Change - This button changes the active program number on the G-Major. This is useful when the real-time option is set to off.
```

> Copy - Copy an existing preset to the selected position. If a preset is present on this location it is destroyed.

Reviewer: ```suggestion
Copy - Copy an existing preset to the selected position. If a preset is present at this location, it is destroyed.
```

Delete - Destroys the selected preset.

Import - Import presets previously exported or downloaded from guitar patches.

> Export - Exports presets as a .realpatch file, which can be dropped onto guitar patches or exchanged with other users.

Reviewer: `dropped onto`

À mon avis incorrect, peut être que `dropped on` convient mieux. À vérifier.

Author: Je pense aussi :-)
Corrigé.

> Here is a link to guitar patches for gourmands&nbsp;:

Reviewer: :smile:

Author: Elle est bonne celle là hein :-)

https://guitarpatches.com/patches.php?unit=GMajor

=== MIDI

The MIDI menu allows you to select the midi-out interface to use. +
The real time checkmark switches to the real-time mode. This mode allows you to apply changes instantly. +

> Send – This button sends the current preset to the G-Major when the real-time mode is not activated. +

Reviewer: Même remarque qu'ailleurs. Cela fonctionne aussi en mode temps réel.

```suggestion
Send – This button sends the current preset to the G-Major. This is useful when the real-time mode is inactive. +
```

> Receive – This button opens a window and put Freemajor in reception mode. Send a Bulk Dump message from the machine and validate after reception ok. Be carrefull : This process overwrite the actual bank. +

Reviewer: Correction fautes + message en cohérence avec logiciel version anglaise.

Remarque 1 : je crains que le mot anglais `reception` soit un faux ami au sens tout autre de ce qu'on croit. (à vérifier)

Remarque 2 : il n'y a pas de notion dans FreeMajor de `mode réception`. J'ai formulé ça d'une autre manière.

```suggestion
Receive – This button opens a window and lets Freemajor listen to incoming preset data. Send a Bulk Dump from the machine and validate when finished. Be careful : This process overwrites the current bank. +
```

> Receive – This button opens a window and lets Freemajor listen to incoming preset data. Send a Bulk Dump from the machine and validate when finished. Be careful : This process overwrites the current bank. +

Reviewer: ```suggestion
Receive – This button opens a window and lets FreeMajor listen to incoming preset data. Send a Bulk Dump from the machine and validate when finished. Be careful: This process overwrites the current bank. +
```

=== Patch parameters

> Here you set the routing, the relays, the tap tempo, the preset volume. To change the name of the current preset, simply write the new name in the patchname field and press enter. In real-time mode, the new name is sent to the G-Major which updates it.

Reviewer: `patchname` → `patch name`
On peut envisager de mettre une emphase sur les références à des éléments de l'interface.

Author: Corrigé :-)
Pour les évolutions futures faudra le faire après la version préliminaire.
Je garde l'idée en tête.

> The 'modifiers' button opens a window which allows to assign a parameter to a 'modifier' and to allow the adjustment of the response curve.

Reviewer: ```suggestion
The 'modifiers' button opens a window which allows to assign a parameter to a 'modifier' and adjust the response curve.
```

> It is in the following form&nbsp;:

Reviewer: ```suggestion
It is structured in the following manner&nbsp;:
```

.Modifiers assignment and adjustment window.
image::../resources/en/modifiers.png[500,500]

=== Parameter infos

> This window displays the description of each switch or potentiometer that allows the effects to be adjusted when hovering with the mouse pointer.

Reviewer: ```suggestion
This window displays the description of each switch or potentiometer that allows the effects to be adjusted when hovered with the mouse pointer.
```

== À propos

> FreeMajor is a software developed by Jean Pierre Cimalando in 2018. +

Reviewer: Mauvais usage de l'article `a` avec `software`.
Aussi, tu as le droit de te réclamer comme co-développeur alors inscris ton nom ici.

```suggestion
FreeMajor is software developed by Jean Pierre Cimalando in 2018. +
```

> This software is made with a goal&nbsp;: facilitate the use of the G-Major multi-effects which can only be controlled by the buttons present on the front panel or by injecting a Sysex message.

Reviewer: ```suggestion
This software is designed with one objective&nbsp;: facilitate the use of the G-Major multi-effects unit which can otherwise only be controlled by the front panel or by injecting a system-exclusive message.
```
